lib/infomon/activespell.rb

```diff
@@ -70,6 +73,39 @@ def self.get_spell_info(spell_check = XMLData.active_spells)
     [update_spell_names, update_spell_durations]
   end
 
+  def self.show_duration_change
+    active_durations = Array.new
+    [Effects::Spells, Effects::Cooldowns, Effects::Buffs, Effects::Debuffs].each do |effect_type|
+      active_durations += effect_type.to_h.keys
+      effect_type.to_h.each do |effect, end_time|
+        next unless effect.is_a?(String)
+        effect, end_time = ActiveSpell.get_spell_info({ effect=>end_time })
+        effect = effect.join
+        end_time = end_time[effect]
+        next unless (spell = Spell.list.find { |s| s.num == effect.to_i || s.name =~ /#{effect}/ })
```

Comment:
Our old friend 620 strikes again: Resist Nature - Heat (22%
--- Lich: error: end pattern with unmatched parenthesis: /Resist Nature - Heat (22%/

A Regexp.escape would fix the crash but it would also not match the spell name, could do need to some spell name fixing like is done in other places.

Reply:
Should be resolved via PR #485 when merged together as this function uses that lookup/name correction.